components/tokens/custom-spectrum/custom-medium-vars.css

```diff
@@ -59,9 +59,7 @@ governing permissions and limitations under the License.
   --spectrum-alert-banner-edge-to-button: var(--spectrum-spacing-100);
   --spectrum-alert-banner-text-to-button-vertical: var(--spectrum-spacing-100);
 
-  --spectrum-sidenav-heading-top-margin: 24px; /* TODO replace with var(--spectrum-side-navigation-item-to-header); once token is updated */
-  --spectrum-sidenav-heading-bottom-margin: 8px; /* TODO replace with var(--spectrum-side-navigation-item-to-header); once token is added */
-  --spectrum-sidenav-bottom-to-label: 8px; /* TODO replace with updated var(--spectrum-component-bottom-to-text-100); */
+  --spectrum-sidenav-bottom-to-label: 8px;
```

Comment:
@jenndiaz are we still using this custom property? When I look at the `index.css` for Side Navigation, I see this: `--spectrum-sidenav-bottom-to-label: var(--spectrum-side-navigation-bottom-to-text);`

Should we remove this from the custom tokens now?

Reply:
yes we should, it is removed now!